src/webui/www/private/scripts/client.js

```diff
@@ -864,6 +873,22 @@ window.addEvent('load', function() {
         registerMagnetHandler();
     });
 
+    $('showFiltersSidebarLink').addEvent('click', function(e) {
+        showFiltersSidebar = !showFiltersSidebar;
```

Comment:
I would prefer you call a function that will get the sidebar state and not use a global variable.

Reply:
Nice catch, it wasn't supposed to be a global variable. Check [this](https://github.com/qbittorrent/qBittorrent/pull/15409#discussion_r700970999).

Reply:
Done